### PR TITLE
Let us no longer create java files

### PR DESCRIPTION
**Pull request by tobiasKaminsky**
Signed-off-by: tobiasKaminsky <tobias@kaminsky.me>

- [x] Tests written, or not not needed

### CODE REVIEW

**Comment by tobiasKaminsky** [comment]
```
Run scripts/analysis/detectNewJavaFiles.sh
fatal: ambiguous argument 'origin/master': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'
```

As checkout action only fetches specific branch, there is no master.

**Comment by tobiasKaminsky** [comment]
works now 💪

File: scripts/analysis/detectNewJavaFiles.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+count=$(grep \.java\" -c $HOME/files_added.json)
+
+if [ $count -eq 0 ] ; then
+    exit 0
+else
+    exit 1
```
Comment (AlvaroBrey) [inline scripts/analysis/detectNewJavaFiles.sh:8]:
```suggestion
    echo "New Java files detected! Please use Kotlin for new files. Number of new Java files: $count"
    exit 1
```

**Review by AlvaroBrey — APPROVED** [review]
Looks good, minor suggestion to add some output to the script

**Pull request merged**